packages/react-core/src/components/DescriptionList/examples/DescriptionList.md

```diff
@@ -120,6 +120,40 @@ import FlagIcon from '@patternfly/react-icons/dist/esm/icons/flag-icon';
 ```ts file='./DescriptionListDefaultInlineGrid.tsx'
 ```
 
+### Display variant
+
+```ts file='./DescriptionListDisplay.tsx'
+```
```

Comment:
Since this example isn't showcasing the inline grid variant, it'd be better served either immediately after the "column fill" example (or somewhere before the "Responsive column definitions" section), or in an entirely new "Display variants" section (could be useful if we expect additional display variants to be imlpemented). 

@mattnolting since this would result in an update in Core, do you have a strong opinion one way or another?

Reply:
@thatblindgeye Agree, these new examples should all be grouped under display variants.

Reply:
Okay, so I created a new `Display variants` section